event-streams/src/event-streams-v1.0-wd01.md

```diff
@@ -683,17 +695,42 @@ Examples:
 The filter-type is “amqp:event-streams-sql-filter”, with type-code 0x00000000:0x00000201
 
 
-
-
 -------
 
 # 6 Runtime Information
 
-TBD.
-
-> This section will describe a request/response operation to acquire information
-> about the available partitions and offsets. 
-
+Each event log node MUST implement a special, subordinate source address named
+`$info` that allows producers and consumers to obtain information about the
+configuration and state of the event log, most importantly the current number of
+partitions and the offset boundaries for each partition.
+
+If the container-relative source address of an event log node is "example", the
```

Comment:
do you mean `event log` here, right? not the node ...

Reply:
I mean the node. The node is the thing that shows up in AMQP.

Reply:
and where do you specify the `event log` for which you want to get partitions?